ui/src/app/components/text-column.tsx

```diff
@@ -3,12 +3,12 @@ import colors from 'app/styles/colors';
 import * as React from 'react';
 
 export const TextColumn = (props) => {
-  return <FlexColumn style={{
+  return <FlexColumn style={{...{
     color: colors.primary,
```

Comment:
I think this may be worth pulling into a style object since you have to spread operator it. Just may be a bit more clear.